src/components/bookProgress.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import { CircularProgressbar, buildStyles } from 'react-circular-progressbar';
+
+const progressbarStyles = buildStyles({
+  pathColor: '#0290ff',
+  trailColor: '#d6d6d6',
+});
+
+const Progress = () => (
+  <>
+    <div className="books-section">
+      <section className="percent-section">
+        <div className="progress-circle">
+          <CircularProgressbar value={64} styles={progressbarStyles} />
+        </div>
+        <div className="percentage">
+          <li className="value">64%</li>
+          <li className="completed">Completed</li>
+        </div>
+      </section>
+      <div className="percent-separator"></div>
+      <section className="chapter-section">
+        <ul className="chapter-details">
+          <li className="current-chapter">CURRENT CHAPTER</li>
+          <li className="chapter-number">Chapter 17</li>
+          <li>
+            <button type="submit" className="update-progress">
+              UPDATE PROGRESS
+            </button>
+          </li>
+        </ul>
+      </section>
+    </div>
```

Comment:
- [ ] Please fix your overlap text from this portion and stick to the [Zaplin](https://zeplin.io/) design :+1: check below :point_down:  the comparison between your design and [Zaplin](https://zeplin.io/) design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-25 03-17-37](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/7eac0a9e-8906-4a45-926e-4f91ad3e1355)

**Zaplin Design**
![Screenshot from 2023-06-25 03-21-58](https://github.com/MoyasiGinko/React-BookStore/assets/45226848/2bad26ac-c29e-4a38-99cf-77faf243447a)

- [ ] Please increase the width of your text then it will look like [Zaplin](https://zeplin.io/) design  :+1:
- [ ] Please change your button color because it's not matching with [Zaplin](https://zeplin.io/) design :+1: